### PR TITLE
add installation guidance in GKE

### PR DESCRIPTION
**Pull request by LillyWuElastisys**
1. Fix deployment part in mainfestr and mainfests
2. add installation guidance in GKE for reference

### CODE REVIEW

File: kube-insight-installation-in-GKE.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+### 1.3. List the cluster information
+```
+li_wu@cloudshell:~ (test1012-219213)$ gcloud container clusters list
+NAME       LOCATION         MASTER_VERSION  MASTER_IP      MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
+insight    europe-north1-a  1.9.7-gke.6     35.228.164.55  n1-standard-1  1.9.7-gke.6   3          RUNNING
+monitored  europe-north1-a  1.9.7-gke.6     35.228.38.59   n1-standard-1  1.9.7-gke.6   3          RUNNINGli_wu@cloudshell:~ (test1012-219213)$
+```
+### 1.4. Enable kubectl and get cluster-info with --cluster specified
+```
+kubectl cluster-info --cluster gke_test1012-219213_europe-north1-a_insight
```
Comment (Xartos) [inline kube-insight-installation-in-GKE.md]: Is it possible to add a snippet before this, that extracts the names of the cluster into a variable that can be used when referring to the different clusters? (haven't worked in GKE so I don't know if it's even possible)

File: kube-insight-installation-in-GKE.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+```
+Error from server (Forbidden): error when creating "output/node-exporter/node-exporter-rbac.yaml": clusterroles.rbac.authorization.k8s.io "node-exporter" is forbidden: attempt to grant extra privileges: [PolicyRule{Resources:["tokenreviews"], APIGroups:["authentication.k8s.io"], Verbs:["create"]} PolicyRule{Resources:["subjectaccessreviews"], APIGroups:["authorization.k8s.io"], Verbs:["create"]}] user=&{li.wu@elastisys.com  [system:authenticated] map[user-assertion.cloud.google.com:[AGKDXmrmxR4ldy6MKeKjP/yU11f3oYRk8PD+N0PgGAq22xLez1St1GzcuML16US1LpSUuGhBOnxyxwZsHp7SCQjDpznRqruCdKWr/VWWNPbAwug79cP3/aWm082ChAydsV4O9mpVKPNRdUeQGWHrVs1nNYADq9+a39CgHjH6lHF0hGtRHFdZ+21irqPHQYoc3/FkERUC2ax9wRdypoU/lgYoRVj9o4siTnwHcmU=]]} ownerrules=[PolicyRule{Resources:["selfsubjectaccessreviews" "selfsubjectrulesreviews"], APIGroups:["authorization.k8s.io"], Verbs:["create"]} PolicyRule{NonResourceURLs:["/api" "/api/*" "/apis" "/apis/*" "/healthz" "/swagger-2.0.0.pb-v1" "/swagger.json" "/swaggerapi" "/swaggerapi/*" "/version"], Verbs:["get"]}] ruleResolutionErrors=[]
+```
+#### 3.1.4 Deployment
+`kubectl apply -R -f output/ --cluster gke_test1012-219213_europe-north1-a_monitored`
```
Comment (Xartos) [inline kube-insight-installation-in-GKE.md]: Somehow this snippet gets merged into the one under? Do you need three ` here as well?

File: kube-insight-installation-in-GKE.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+```
+(manifestr)li_wu@cloudshell:~/kube-insight-manifests/manifests/agents/logging (test1012-219213)$ manifestr --values `pwd`/values.py --template-root-dir `pwd`/templates --output-dir `pwd`/output
+```
+#### 3.2.2.  Create namespace logging
+`kubectl create ns logging  --cluster gke_test1012-219213_europe-north1-a_monitored`
+
+#### 3.2.3. Deployment
+```
+(manifestr)li_wu@cloudshell:~/kube-insight-manifests/manifests/agents/logging (test1012-219213)$ kubectl apply -R -f output/ --cluster gke_test1012-219213_europe-north1-a_monitored
+After deployment,  pods are:
```
Comment (Xartos) [inline kube-insight-installation-in-GKE.md:110]: Maybe split this into two snippets?

File: kube-insight-installation-in-GKE.md (hunk 1 above)
Comment (LillyWuElastisys) [reply kube-insight-installation-in-GKE.md]: This is the snippet to get the cluster name. To extract the name, there should be a way, but wouldn't easier and simple single command.

File: kube-insight-installation-in-GKE.md (hunk 3 above)
Comment (LillyWuElastisys) [reply kube-insight-installation-in-GKE.md:110]: Thank you for your careful review, I just split it.

File: kube-insight-installation-in-GKE.md (hunk 1 above)
Comment (Xartos) [reply kube-insight-installation-in-GKE.md]: Perfect! At least now one who knows noting is able to fetch the name :stuck_out_tongue:

File: kube-insight-installation-in-GKE.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
+# kube-insight installation in Google Kubernetes Engine
+
+kube-insight repository:  https://github.com/elastisys/kube-insight-manifests
+## 1.Setup two clusters in GKE
+According to the kube-insight recommendation, Creating two clusters: one named monitored(agent) and the other named insight(server)
+
+Follow the official doc:  https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster
+### 1.1. Set zone and region
+```
+li_wu@cloudshell:~ (test1012-219213)$ gcloud config set compute/zone europe-north1-a
```
Comment (petergardfjall) [inline kube-insight-installation-in-GKE.md]: The shell prompt brings a lot of noise to each command row. I would advice to make it shorter. For instance turn

`li_wu@cloudshell:~ (test1012-219213)$ gcloud config set compute/zone europe-north1-a`

into 

`$ gcloud config set compute/zone europe-north1-a`

Comment (petergardfjall) [reply kube-insight-installation-in-GKE.md]: ... and do this on all occurrences of running a command from the prompt

**Comment by petergardfjall** [comment]
I have no reservations beyond my comment about the prompt. Fix those and then the merge is good to go from my point of view.

**Comment by LillyWuElastisys** [comment]
@petergardfjall  Fixed the noise in prompt.

**Comment by petergardfjall** [comment]
@LillyWuElastisys Great, merging...

**Pull request merged**